views/index.ejs

```diff
@@ -10,8 +10,10 @@
 
     <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
     <ul>
-      <!-- List the names of the items in your cart here -->
 
+    <% cart.forEach(function(product)) { %>
```

Comment:
I think your syntax is a little off here - you want this to be:

```js
<% cart.forEach(function(product) { %>
  <li><% product.name %></li>
<% }); %>
```

so that the closing parenthesis on the `forEach` function actually wraps the entire callback function.